libexec/bats

```diff
@@ -20,6 +20,8 @@ help() {
   echo "  -h, --help     Display this help message"
   echo "  -p, --pretty   Show results in pretty format (default for terminals)"
   echo "  -t, --tap      Show results in TAP format"
+  echo "  -x, --extended Show results in extended TAP format"
+  echo "  -u, --junit    Show results in JUnit format, hudson compatible"
```

Comment:
I think it would be more explicit and future-proof to name them `-x, --extended-tap` and `-j, --junit` .